packages/frontend/app/clipboard-history/page.tsx

```
@@ -3,20 +3,161 @@
+                switch (type) {
+                    case "text":
+                    case "colour":
+                        return {
+                            hash,
+                            type,
+                            text: value.toLowerCase(),
+                        }
+                    case "html":
+                    case "url":
+                    case "path":
+                        const {metadata: { text }} = item
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)